aiida_quantumespresso/workflows/pw/bands.py

```diff
@@ -110,6 +112,46 @@ def define(cls, spec):
             help='The computed band structure.')
         # yapf: enable
 
+    @classmethod
+    def get_builder_from_protocol(cls, code, structure, protocol=None, overrides=None, **kwargs):
+        """Return a builder prepopulated with inputs selected according to the chosen protocol.
+
+        :param code: the ``Code`` instance configured for the ``quantumespresso.pw`` plugin.
+        :param structure: the ``StructureData`` instance to use.
+        :param protocol: protocol to use, if not specified, the default will be used.
+        :param overrides: optional dictionary of inputs to override the defaults of the protocol.
+        :param kwargs: additional keyword arguments that will be passed to the ``get_builder_from_protocol`` of all the
+            sub processes that are called by this workchain.
+        :return: an process builder instance with all inputs defined ready for launch.
+        """
+        args = (code, structure, protocol)
+        inputs = cls.get_protocol_inputs(protocol, overrides)
+        builder = cls.get_builder()
+
+        relax = PwRelaxWorkChain.get_builder_from_protocol(*args, overrides=inputs.get('relax', None), **kwargs)
```

Comment:
So, if one would want to do a band structure calculation without first doing a geometry optimization, this would be the easiest:

```Python
builder = PwBandsWorkChain.get_builder_from_protocol(
    code=code,
    structure=structure,
)
builder.pop('relax')
```
I suppose this can't be specified via the `overrides`?

Reply:
Yes, that would be the way. I don't see a nice way to allow to specify this idea through the overrides.